odkx-src/data-permission-filters.rst

```diff
@@ -427,7 +427,7 @@ You will be presented with a demo launch screen.
 
 At this point, all the rows in all the tables have a :th:`_sync_state` of :tc:`new_row` and are fully editable and deletable. The demo will not become interesting until you set up and sync with a server.
 
-Set up an ODK-X Cloud Endpoint or ODK Aggregate 1.4.15 server with 2 ordinary users, 1 super-user and 1 tables administrator. :guilabel:`Reset App Server` to push the configuration and data up to the server.
```

Comment:
I would not drop the number 1.4.15 as that is the last time we tested it. If someone wants to know what was the full test. :)

Reply:
The 1.4.15 is listed with the Tables Extension link -- I am just deleting here (permission filters) so as to discourage folks from using Aggregate. Can reverse if it's important to have here.

Reply:
It is not as long as it was listed somewhere. :)